engine/lib/upgrades/2015041400-1.11.0_dev-trim_metastrings-d9a9fdfa28a981a3.php

```diff
@@ -0,0 +1,31 @@
+<?php
+/**
+ * Elgg 1.11.0-dev upgrade 2015041400
+ * trim_metastrings
+ *
+ * Trim all whitespace padding in metastrings
+ */
+
+$prefix = elgg_get_config('dbprefix');
+$q = "SELECT * FROM {$prefix}metastrings WHERE string REXEXP '^[ ]+|[ ]+$'";
+$r = mysql_query($q);
+
+while ($ms = mysql_fetch_assoc($r)) {
```

Comment:
Yes.

On Wed, Apr 15, 2015 at 8:10 AM, Steve Clay <notifications@github.com>
wrote:

> In
> engine/lib/upgrades/2015041400-1.11.0_dev-trim_metastrings-d9a9fdfa28a981a3.php
> <https://github.com/Elgg/Elgg/pull/8204#discussion_r28413690>:
>
> > @@ -0,0 +1,31 @@
> > +<?php
> > +/**
> > + * Elgg 1.11.0-dev upgrade 2015041400
> > + * trim_metastrings
> > + *
> > + * Trim all whitespace padding in metastrings
> > + */
> > +
> > +$prefix = elgg_get_config('dbprefix');
> > +$q = "SELECT * FROM {$prefix}metastrings WHERE string REXEXP '^[ ]+|[ ]+$'";
> > +$r = mysql_query($q);
> > +
> > +while ($ms = mysql_fetch_assoc($r)) {
>
> Are you using ext/mysql directly to avoid pulling the whole result set
> into memory?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Elgg/Elgg/pull/8204/files#r28413690>.
>



-- 
Brett Profitt
Elgg Lead Developer

Elgg: http://elgg.org/
Skype: brett.profitt
Twitter: http://twitter.com/brettprofitt